usecase/parser/msg.go

```diff
@@ -1289,22 +1494,30 @@ func ParseMsgRevoke(
 			RawMsgRevoke: rawMsg,
 		}
 
+		// Getting possible signer address from Msg
+		var possibleSignerAddresses []string
+		possibleSignerAddresses = append(possibleSignerAddresses, revokeParams.Granter)
```

Comment:
may i know where do we assign `Granter` to `revokeParams`?

Reply:
The `RawMsgRevoke` type got `Granter`
https://github.com/crypto-com/chain-indexing/blob/e6d137d63ffe8b75710ea64b356798e1219b5a3b/usecase/parser/msg.go#L1497

Reply:
looks like `revokeParams. Granter` must be `nil`?

Reply:
so using `revokeParams.RawMsgRevoke.Granter` instead, right?

Reply:
`revokeParams.RawMsgRevoke.Granter` make more sense. And i saw other parsers might have similar situation, can you help to double check?

Reply:
yea, sure